### PR TITLE
Fixing PR13731 shader error on Mac

### PR DESCRIPTION
**Pull request by samcake**
Same as https://github.com/highfidelity/hifi/pull/13731
With the fix of the shader failing on Mac

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/12003/HighFidelity-Beta-PR13800-5fd3862.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12003/HighFidelity-Beta-PR13800-5fd3862.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12003/HighFidelity-Beta-Interface-PR13800-5fd3862.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/12003/HighFidelity-Beta-PR13800-5fd3862.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/12003/HighFidelity-Beta-Interface-PR13800-5fd3862.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/12013/HighFidelity-Beta-PR13800-cd7ca96.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12013/HighFidelity-Beta-PR13800-cd7ca96.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12013/HighFidelity-Beta-Interface-PR13800-cd7ca96.dmg

File: libraries/render-utils/src/RenderShadowTask.cpp [hunk 1]
```diff
@@ -33,6 +33,89 @@ using namespace render;
 
 extern void initZPassPipelines(ShapePlumber& plumber, gpu::StatePointer state);
 
+void RenderShadowTask::configure(const Config& configuration) {
+    DependencyManager::get<DeferredLightingEffect>()->setShadowMapEnabled(configuration.enabled);
+    // This is a task, so must still propogate configure() to its Jobs
+    //    Task::configure(configuration);
```
[the remark this answers is not in the document]
Comment (samcake) [reply libraries/render-utils/src/RenderShadowTask.cpp]: it s a comment that looks like  code

**Comment by qaAdamSmith82** [comment]
Currently Testing

**Comment by qaAdamSmith82** [comment]
Fail @conklin94122 

I am failing this for now, because I am about to jump in a two hour meeting, and I want to give someone time to look at it during that time

Failing for two reasons:
1.  Clicking Show Views and Show Proxies from the Workload button causes a significant performance hit that did not exist when those tests were run in 13731
2. A pretty significant failure for one of the Auto-Tester shadow runs.  It's so significant I am inclined to believe its user error on my part, so I am working on a command on Mac to launch interface, pull down the script and put the user where they need to be to properly run the test to ensure it is not user error.  I am almost done, and will retest that later today.
tests engine render shadows primarycamera front 00000
tests engine render shadows primarycamera front 00000_ei

**Comment by qaAdamSmith82** [comment]
Pass @conklin94122 

Sam and I were able to determine the test failures were related to the automation scripts themselves, and the performance drop is not a blocking issue

**Pull request merged**